test/e2e/storage/vsphere/vsphere_utils.go

```diff
@@ -756,3 +757,72 @@ func GetReadySchedulableRandomNodeInfo() *NodeInfo {
 	Expect(nodesInfo).NotTo(BeEmpty())
 	return nodesInfo[rand.Int()%len(nodesInfo)]
 }
+
+// invokeVCenterServiceControl invokes the given command for the given service
+// via service-control on the given vCenter host over SSH.
+func invokeVCenterServiceControl(command, service, host string) error {
+	sshCmd := fmt.Sprintf("service-control --%s %s", command, service)
+	framework.Logf("Invoking command %v on vCenter host %v", sshCmd, host)
+	result, err := framework.SSH(sshCmd, host, framework.TestContext.Provider)
```

Comment:
`BeforeEach` does not explicitly test for ssh access but this may be implied by verifying that the provider is vshpere/

Reply:
That's correct.